src/components/Countdown/Countdown.tsx

```diff
@@ -76,13 +81,17 @@ const Countdown = forwardRef<HTMLDivElement, CountdownProps>(
       <div {...commonProps} className={classnames(baseClassName, className)} {...props} ref={ref}>
         <div className={`${baseClassName}__countdown-container`} role="timer" aria-label={label}>
           <span>{label}</span>
-          {timeLeft.days > 0 ? <Duration duration={timeLeft} unit="days" locale={dateFnsLocale} /> : undefined}
+          {timeLeft.days > 0 ? (
+            <Duration duration={timeLeft} unit="days" locale={dateFnsLocale} formatDurationStr={formatDurationStr} />
+          ) : undefined}
```

Comment:
I think we are using null instead of undefined for conditional rendering, even though undefined is allowed with new react version, null is better for our consistency